rpcd/etc/openstack_deploy/user_osa_variables_defaults.yml

```diff
@@ -142,6 +142,12 @@ repo_build_upper_constraints_overrides:
   # the optional dependencies specfied here:
   # https://github.com/openstack/ceilometer/blob/stable/newton/setup.cfg#L36
   - "gnocchiclient>=2.2.0,<3.0.0"
+  # The elasticsearch version is pinned here due to a bug within the Ansible
+  # pip module. The Ansible pip module can no longer accept constraints as
+  # part of the 'name' parameter. To work around this, the constraint is added
+  # here then passed to pip using the "extra_args" parameter.
+  # https://github.com/ansible/ansible/issues/22399
+  - "elasticsearch>=2.0.0,<3.0.0"
```

Comment:
This is an entirely unnecessary pin as OpenStack already has a pin for elasticsearch: https://github.com/openstack/requirements/blob/stable/ocata/upper-constraints.txt#L129